CMakeLists.txt

```diff
@@ -0,0 +1,213 @@
+cmake_minimum_required(VERSION 3.12)
+cmake_policy(SET CMP0076 NEW)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/sys/cmake")
+include(CMakeDependentOption)
+include(ucm)
+include(PrecompiledHeader)
+include(SvnVersion)
+
+project(TheDarkMod VERSION 2.08)
```

Comment:
I wonder if this has any meaning.
It is probably easier to not set any version, since otherwise there is always risk of forgetting to update the version somewhere =)

Reply:
Yeah, probably. This could make sense if CMake were the only build system and could be the actual source of truth for the version, but otherwise it's just going to get out of sync :)